src/components/PrimaryButton.jsx

```diff
@@ -0,0 +1,11 @@
+import './PrimaryButton.css';
+
+function PrimaryButton({ children, onClick, disable = false }) {
```

Comment:
P2:
컴포넌트 속성명은 가능하면 tag의 속성과 동일하게 쓰는 것이 사용성에서 더 좋습니다.

```suggestion
function PrimaryButton({ children, onClick, disabled = false }) {
```